demo/lib/vars/testInParallel.groovy

```diff
@@ -13,9 +13,7 @@ def call(parallelism, inclusionsFile, exclusionsFile, results, image, prepare, r
             writeFile file: (split.includes ? exclusionsFile : inclusionsFile), text: ''
           }
           stage('Main') {
-            realtimeJUnit(results) {
```

Comment:
Then maybe it is time to create a separate demo, or put this on hold until `realtimeJUnit` is made compatible.